orttraining/orttraining/python/training/torchdynamo/ort_backend.py

```diff
@@ -59,13 +60,14 @@ def _nvtx_range_pop():
         torch.cuda.nvtx.range_pop()
 
 
-def _get_ort_device_type(device_type: str, device_index: int):
+def _get_ort_device_type(device_type: str, device_kind: int):
     if device_type == "cuda":
         return ORTC.OrtDevice.cuda()  # type: ignore
     if device_type == "cpu":
         return ORTC.OrtDevice.cpu()  # type: ignore
+    # fall back to OrtBackend configuration
     if device_type == "ort":
-        return ORTC.get_ort_device(device_index).device_type()  # type: ignore
+        return device_kind
```

Comment:
i don't think we has plan to map torch 'ort' device key to multiple device types in ORT. so i don't think we need the flexibility to make it configurable. it seems not worth to expose it in API.

Reply:
'ort' dispatch key is unrelated to onnxruntime and the device type for it should be considered as an unique type in ORT as you mentioned. However, since the device is temporarily borrowing a device type in ORT (FPGA device type), I left this part to be configured externally by setting device kind for mapping 'ort' to ort device type. 

Does it make sense to make it return an enum (e.g. FPGA) for 'ort' device type in this PR then later fix it to a device type?

Reply:
Created [PR14755](https://github.com/microsoft/onnxruntime/pull/14755) to discuss the part.

Reply:
Removed device device config for OrtBackend, and added mapping for ort dispatch key to NPU ort device type internally. Closing the comment.